diff -u -r -U5 git-annex-0.11.install-docs/Makefile git-annex-0.11.prefix-support/Makefile
--- git-annex-0.11.install-docs/Makefile	2010-12-08 22:23:32.755992078 -0500
+++ git-annex-0.11.prefix-support/Makefile	2010-12-08 22:24:25.444992024 -0500
@@ -1,5 +1,7 @@
+PREFIX=/usr/local
+
 all: git-annex docs
 
 ghcmake=ghc -Wall -odir build -hidir build -O2 --make 
 
 SysConfig.hs: configure.hs
@@ -8,18 +10,18 @@
 
 git-annex: SysConfig.hs
 	$(ghcmake) git-annex
 
 install:
-	install -d $(DESTDIR)/usr/bin
-	install git-annex $(DESTDIR)/usr/bin
+	install -d $(DESTDIR)$(PREFIX)/bin
+	install git-annex $(DESTDIR)$(PREFIX)/bin
 
 install-docs:
-	install -d $(DESTDIR)/usr/share/man/man1/
-	install git-annex.1 $(DESTDIR)/usr/share/man/man1/git-annex.1
-	install -d $(DESTDIR)/usr/share/doc/git-annex/
-	cp -a html $(DESTDIR)/usr/share/doc/git-annex/html
+	install -d $(DESTDIR)$(PREFIX)/share/man/man1/
+	install git-annex.1 $(DESTDIR)$(PREFIX)/share/man/man1/git-annex.1
+	install -d $(DESTDIR)$(PREFIX)/share/doc/git-annex/
+	cp -a html $(DESTDIR)$(PREFIX)/share/doc/git-annex/html
 
 # If ikiwiki is available, build static html docs suitable for being
 # shipped in the software package.
 ifeq ($(shell which ikiwiki),)
 IKIWIKI=@echo "** ikiwiki not found, skipping building docs" >&2; true
